### ISSUE TITLE
Have a `TabKindCustomDiff`

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#145743]
Testing #145585

currently if I compare 2 images, there is no `kind` then.

**Comment by jrieken** [comment]
Kind of related to https://github.com/microsoft/vscode/issues/145688 because a diff of custom editors isn't really a diff editor, e.g it doesn't show differences but two resources next to each other

**Comment by bpasero** [comment]
Yes you are right, we could solve this one with the other one if we had it.

**Comment by lramos15** [comment]
> Kind of related to #145688 because a diff of custom editors isn't really a diff editor, e.g it doesn't show differences but two resources next to each other

This is up to the implementation of the custom editors. It could in theory render a diff if I remember correctly @mjbvz

**Comment by mjbvz** [comment]
Yes custom editors always render in diffs today if they are configured a the default editor for that file type. We may relax this at some point however and let custom editors tell us to use text editors for diffs instead
